src/handlers/motions/motionStaked/helpers.ts

```diff
@@ -55,10 +57,13 @@ export const getRemainingStakes = (
 export const getStakePercentage = (
   stake: BigNumber,
   requiredStake: BigNumber,
+  roundToOne: boolean,
```

Comment:
This could have a default param set to `true` so you don't need to pass it explicitly below